app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,7 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <%= render 'favicon' %>
   <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'key_press_listener' %>
```

Comment:
It should be required in the application.js so we don't need to include it here.
You can move it to the `application` folder 
> //= require_tree ./application/

or you can explicitly import in the `application.js` file